README.md

```diff
@@ -23,6 +23,17 @@ Short is built on top of [app](https://github.com/byliuyang/app), a reusable fra
 ### Create reCAPTCHA account
 [Create ReCAPTCHA account](http://www.google.com/recaptcha/admin)
 
+Register at [ReCAPTCHA](http://www.google.com/recaptcha/admin) using the following configuration.
+|Field       |Value      |
+|--------------|-----------|
+|Label         |short      |
+|reCAPTCHA type|reCAPTCHAv2|
+|Domains       |localhost  |
```

Comment:
I assume localhost works well enough for this but if there is something else you'd like to have here please let me know.